### PR TITLE
[TASK] CGL - Remove multiple variable assignment, fix indents and exceptions

### PR DESCRIPTION
**Pull request by xf-**

### CODE REVIEW

File: Classes/View/LegacyPreviewView.php [hunk 1]
```diff
@@ -54,17 +54,17 @@ class LegacyPreviewView extends PreviewView {
...
-     * @param $colPosFluxContent
-     * @param $dblist
-     * @param $target
-     * @param $id
-     * @param $content
-     * @return string
-     */
+	 * @param array $row
+	 * @param Column $column
+	 * @param $colPosFluxContent
```
Comment (NamelessCoder) [inline Classes/View/LegacyPreviewView.php]: While you are doing CGL anyway: please add parameter types for this and the next handful of arguments.

Comment (xf-) [inline Classes/View/LegacyPreviewView.php]: thx, variables already included, param still missing

Comment (NamelessCoder) [inline Classes/View/LegacyPreviewView.php]: What I mean is that the `@param $colPosFluxContent` should be `@param integer $colPosFluxContent`, `@param $dblist` should be `@param DatabaseRecordList $dblist` etc.

**Comment by xf-** [comment]
Done

**Pull request merged**